evergreen.yml

```diff
@@ -0,0 +1,74 @@
+buildvariants:
+  - display_name: Ubuntu 18.04
+    name: ubuntu1804
+    run_on:
+      - ubuntu1804-small
+    tasks:
+      - name: unit_tests
+
+
+
+functions:
+  create virtualenv:
+    - command: shell.exec
+      params:
+        working_dir: ${git_project_directory|src}
```

Comment:
I don't know why the previous conversation isn't showing up anymore, but please change this to just `${src}` unless we have a good reason for not doing that. 

Also, it is usually not a good idea to resolve other people's comments.

Reply:
Change it back, sorry for resolving those comments. 
I got this idea from mongo repo. I was trying to use bash, and set the working_dir this way could handle case where "src" is git_project_directory and another "src" under "evg-module-manager".